### PR TITLE
added option to suppress warning about empty directory

### PR DESCRIPTION
**Pull request by enikao**
See #787 

I couldn't get the build fully working locally. It _should_ work ...

### CODE REVIEW

File: org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java [hunk 1]
```diff
@@ -115,6 +115,17 @@ public void suppressWarningsAnnotation() throws Exception {
 		assertFileDoesNotContain(verifier, outputdir + "src/test/generated-sources/xtend/test/XtendB.java", "@SuppressWarnings");
 	}
 
+	@Test
+	public void suppressEmptyDirWarning() throws Exception {
+		Verifier verifier = MavenVerifierUtil.newVerifier(ROOT + "/emptydir-warning");
+		System.out.println(verifier.getLogFileName());
```
Comment (ArneDeutsch) [inline org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java]: Leftover to be removed befiore merge!?

**Comment by cdietrich** [comment]
@enikao please have a look at your ECA https://www.eclipse.org/legal/ECA.php

File: org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java (hunk 1 above)
Comment (enikao) [reply org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java]: Not really, I copied the code from `suppressWarningsAnnotation()`.
We could still remove it, though. What do you think?

**Comment by cdietrich** [comment]
@ArneDeutsch what is the review state for this pr?

File: org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java [hunk 2]
```diff
@@ -115,6 +115,17 @@ public void suppressWarningsAnnotation() throws Exception {
 		assertFileDoesNotContain(verifier, outputdir + "src/test/generated-sources/xtend/test/XtendB.java", "@SuppressWarnings");
 	}
 
+	@Test
+	public void suppressEmptyDirWarning() throws Exception {
```
Comment (cdietrich) [inline org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java]: i wonder if we should add a negative test too

Comment (szarnekow) [reply org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java]: +1

**Comment by cdietrich** [comment]
@enikao can you please remove the sysouts and add a negative test. thanks

**Comment by enikao** [comment]
> @enikao can you please remove the sysouts and add a negative test. thanks
> 
>     * rebase

Will do, maybe not this week.

**Comment by enikao** [comment]
Shall I remove the sysouts only in my code or also in the other test methods in the same class?

**Comment by cdietrich** [comment]
please remove it in yours only. will have a look at others later

File: org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java (hunk 1 above)
Comment (ArneDeutsch) [reply org.eclipse.xtend.maven.plugin/src/test/java/org/eclipse/xtend/maven/XtendCompilerMojoIT.java]: Yes, I think it should be removed.

**Comment by cdietrich** [comment]
thx @enikao for the fix

**Pull request merged**